### DISCUSSION TITLE
missing 36.32.0 and 36.33.0 in npm registry

### DISCUSSION THREAD

**Discussion by chenrui333** [renovatebot/renovate#23718] [Request Help] [answered]
### Describe the problem

missing 36.32.0 and 36.33.0 in npm registry

![image](https://github.com/renovatebot/renovate/assets/1580956/ef177e77-a9b9-417c-a8fb-56379086ad7e)

**Comment by viceice** [comment, marked as answer]
probably broken release 🤷‍♂️

**Comment by chenrui333** [comment]
covered by https://github.com/Homebrew/homebrew-core/pull/138840
